website/markdown/docs/contribution/website.mdx

```diff
@@ -0,0 +1,26 @@
+---
+name: Website
+excerpt: Learn about Tuist website, how to run it locally
+---
+
+# Contributing
+
+Tuist's website is built using [Gatsby](https://www.gatsbyjs.com/).
+
+### Installing dependencies
+
+Make sure you are on the correct version of Ruby and NodeJS, which are respectively declared on the `.ruby-version` and `.nvmrc` files.
+
+To install the packages needed to for the website, run:
+
+```
+yarn install
+```
+
+### Run the website locally
+
+To deploy Tuist's website on localhost, run:
+
+```
+gatsby develop
```

Comment:
```suggestion
yarn run develop
```

Note that if you call `gatsby` directly, you are running whatever version of Gatsby you have installed globally.

Reply:
good point, thanks